src/jvm/main/org/jetbrains/kotlinx/lincheck/strategy/managed/TracePoint.kt

```diff
@@ -233,6 +244,10 @@ internal class CoroutineCancellationTracePoint(
     }
 }
 
+internal class SpinCycleStartTracePoint(iThread: Int, actorId: Int, callStackTrace: CallStackTrace): TracePoint(iThread, actorId, callStackTrace) {
+    override fun toStringImpl() =  "// The following events repeat infinitely"
```

Comment:
<img width="507" alt="Screenshot 2023-06-30 at 17 13 43" src="https://github.com/JetBrains/lincheck/assets/31644752/0e607bea-ec8b-4db8-94af-9491be494ea3">

IDEA interprets `/` as a path separator
How about `[The following events repeat infinitely:]`?

Reply:
I suggest to use well-known comment syntax: 
`// The following events repeat infinitely:`